api/v1alpha1/eventing_types.go

```diff
@@ -14,6 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// +kubebuilder:validation:Optional // This sets 'required' as the default behaviour.
```

Comment:
The comment does not align with the annotation semantics.
Did you mean `+kubebuilder:validation:Required`?

Reply:
desired default behaviour is `optional`, but I removed the line because `omitempty` takes care of making a field default